tests/e2e/pact/services/convert-example-fetcher.ts

```diff
@@ -0,0 +1,47 @@
+import path from "path"
+import { ConvertCase } from "../models/cases/convert-case"
+import { exampleFiles } from "./example-files-fetcher"
+
+const examplesRootPath = "../resources/parent-prescription"
+
+const convertResponseFiles = exampleFiles.filter(exampleFile => exampleFile.isResponse && exampleFile.endpoint === "convert")
+
+const convertRequestFiles = exampleFiles.filter(exampleFile =>
+	exampleFile.isRequest
+	&& convertResponseFiles.some(convertResponseFile => 
+			convertResponseFile.dir === exampleFile.dir
+			&& convertResponseFile.operation === exampleFile.operation
+			&& convertResponseFile.number === exampleFile.number))
+
+const conventionBasedConvertExamples: ConvertCase[] = convertResponseFiles.map(convertResponseFile => new ConvertCase(
```

Comment:
Same here, could do with cleaning up these arguments with some helper functions to make it more readable?